frontend/src/bundles/overview/components/chart-goal-progress/helpers/convert-stats-to-percentage.helper.ts

```diff
@@ -0,0 +1,33 @@
+import { type ChartGoalDataset } from '../types/types.js';
+
+const defaultValue = 0;
+
+const convertStatsToPercentage = (
```

Comment:
so as 100% will be displayed largest by steps/kkal on timeframe?

Reply:
Yes